test/detect-non-literal-require.js

```diff
@@ -1,16 +1,25 @@
 'use strict';
 
 const RuleTester = require('eslint').RuleTester;
-const tester = new RuleTester();
+
+const tester = new RuleTester({
```

Comment:
We don’t need Babel-eslint for this. You can just specify ecmaVersion: 6 in the tests, as in:

https://github.com/eslint/eslint/blob/79278a14f1c8747bff8f5cb2100d8776f9d517f2/tests/lib/rules/no-use-before-define.js#L31

Reply:
Good catch - thanks